src/main/scripts/cd/external_build.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+
+
+import requests
+from urllib.parse import urljoin
+import json
+import time
+import sys
+import os
+
+class TravisBuilds:
+    """
+    This class provides requests to builds and check their statuses
+    """
+    travis_api_url = 'https://api.travis-ci.com/'
+    build_id = None
+
+    def __init__(self, repo_name, auth_token):
+        self.headers = {'Content-Type': 'application/json',
+                        'Accept': 'application/json',
+                        'Travis-API-Version': '3',
+                        'Authorization': 'token {}'.format(auth_token)
+                        }
+        self.repo_name = repo_name
+
+    def start_build(self):
+        data = {"request": {
+            "branch": "master"
+        }}
+        url = urljoin(self.travis_api_url,
+                      'repo/{}/requests'.format(self.repo_name))
+        response = requests.post(url=url, data=json.dumps(data), headers=self.headers)
+        if response.status_code == 202:
+            self.build_id = self.get_build_id(response.json()["request"]["id"])
+            print(self.build_id)
+        return True
+
+    def get_build_id(self, request_id):
+        time.sleep(10)
+        url = urljoin(self.travis_api_url,
+                      'repo/{}/request/{}'.format(self.repo_name, request_id))
+        response = requests.get(url=url, headers=self.headers)
+        return response.json()["builds"][0]['id']
+
+    def wait_for_build_result(self):
+        attempts = 0
+        tests_minutes = int(os.getenv('TESTS_MINUTES'))
+        while attempts < tests_minutes:
+            url = urljoin(self.travis_api_url, 'build/{}'.format(self.build_id))
+            response = requests.get(url=url, headers=self.headers)
+            if response.json()['state'] == "passed":
+                return True
+            else:
+                print("External build is running {} minutes".format(attempts))
+                time.sleep(60)
+                attempts += 1
+        return False
+
+
+if __name__ == '__main__':
+    external_build = os.getenv('TRIGGER_EXTERNAL_CI', '')
+    if external_build:
+        travis = TravisBuilds(external_build, os.getenv('TRAVIS_AUTH_TOKEN'))
```

Comment:
`TRAVIS_AUTH_TOKEN` should be set on this project in travis, and copied via the [entrypoint script](https://github.com/scalecube/scalecube-travis-ci/blob/master/entrypoint.sh#L63)

Reply:
We need 2 tokens - for org and com

Reply:
used only one for org. this is used in scalecube only

Reply:
fixed via [this commit](https://github.com/scalecube/scalecube-travis-ci/pull/24/commits/b2024549288aa3b38aa089cd408cf6aeaa22437a)